examples/cockroachdb/cockroachdb-statefulset.yaml

```diff
@@ -66,6 +78,21 @@ spec:
       labels:
         app: cockroachdb
       annotations:
+        scheduler.alpha.kubernetes.io/affinity: >
```

Comment:
pod affinity annotation should be added to pod template, not to stateful set, check right example here:
https://github.com/kubernetes/kubernetes/issues/39310#issuecomment-270312557